### ISSUE TITLE
Remove-NavContainer $containerName error

### ISSUE THREAD

**Issue by StijnBt** [microsoft/navcontainerhelper#797]
Hello,

I'm experiencing the error below since upgrading to the latest navcontainerhelper version.

Creating container from SQL backup...
NavContainerHelper is version 0.6.4.24
NavContainerHelper is running as administrator
Host is Microsoft Windows 10 Enterprise - 1909
Docker Client Version is 19.03.5
Docker Server Version is 19.03.5
Using image mcr.microsoft.com/businesscentral/onprem:14.6.36463.0-be-ltsc2019
Disabling Health Check (always report healthy)
Setting host.docker.internal to 192.168.1.109 in container hosts file (copy from host hosts file)
Setting gateway.docker.internal to 192.168.1.109 in container hosts file (copy from host hosts file)
Setting kubernetes.docker.internal to 127.0.0.1 in container hosts file (copy from host hosts file)
The term 'ipconfig' is not recognized as the name of a cmdlet, function, script file, or operable program. 
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

```
At C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.6.4.24\ContainerHandling\New-NavContainer.ps1:521 char:5
Remove-NavContainer $containerName
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
CategoryInfo          : ObjectNotFound: (ipconfig:String) [Remove-NavContainer], CommandNotFoundException
FullyQualifiedErrorId : CommandNotFoundException,Remove-NavContainer
```

**Comment by freddydk** [comment]
if you in a command prompt write: ipconfig
what happens?
and if you do the samem in PowerShell - what happens?

**Comment by StijnBt** [comment]
Hello,

Works in cmd, but only in "run as administrator" mode.
Doesn't work in Powershell

**Comment by freddydk** [comment]
Is PowerShell running in administrator mode?

**Comment by freddydk** [comment]
ipconfig normally works witrhout administrator mode - do you have any special company policies disallowing ipconfig or antivirus software or like?

**Comment by StijnBt** [comment]
Not to my knowledge.
I added %systemroot%\system32 to the System Variables PATH and it worked again as before.
Strange how this stopped working just overnight.

So a coincidence this just happened after the latest navcontainerhelper version.
My apologies.

**Comment by freddydk** [comment]
I would assume that system32 would always be in the PATH - but maybe I should run ipconfig from the system32 path specifically to avoid other occurances of this.

**Comment by freddydk** [comment]
Investigating further because remove-navcontainer shouldn't call ipconfig at all.
I found out that if you remove a container with version 0.6.4.24, which was created with an earlier version, it would call ipconfig and actually not remove the entry in hosts.
I will fix this issue in the next version.

**Comment by freddydk** [comment]
Shipped in 0.6.4.25
